conformance/lang/expressions/unary-expression/unary-complement-expression.balt

```diff
@@ -0,0 +1,515 @@
+Test-Case: output
+Description: Test unary complement basic operations.
+Labels: unary-complement, HexIntLiteral
+
+public function main() {
+    io:println(~0); // @output -1
+
+    int a1 = ~1;
+    io:println(a1); // @output -2
+
+    io:println(~0x0FFF); // @output -4096
+
+    io:println(~~0 == 0); // @output true
+}
+
+Test-Case: output
+Description: Test unary complement operator's operand and result static types.
+Labels: unary-complement
+
+public function main() {
+    io:println(getComplement(10)); // @output -11
+}
+
+public function getComplement(int i) returns int {
+    return ~i;
+}
+
+Test-Case: output
+Description: Test unary complement's bit inversion.
+Labels: additive-expr, unary-complement, unary-minus
+
+public function main() {
+    io:println(~5 == getTwosComplementRepresentation(5)); // @output true
+    io:println(~-5 == getTwosComplementRepresentation(-5)); // @output true
+    io:println(~0 == getTwosComplementRepresentation(0)); // @output true
+}
+
+function getTwosComplementRepresentation(int x) returns int {
+    return -(x + 1);
+}
+
+Test-Case: output
+Description: Test unary complement result type.
+Labels: is-expr, unary-complement
```

Comment:
Shall we make the main construct we are testing the first label everywhere?

```suggestion
Labels: unary-complement, int, is-expr
```

Reply:
I was under the impression we were sticking to alphabetical order. will update

Reply:
I think we need to revisit this order anyway, so let's keep it as is for now maybe?